### PR TITLE
Lesson 1: Javascript, Add/Test

### PR DESCRIPTION
**Pull request by seemcat**

### CODE REVIEW

**Review by allopez7 — APPROVED** [review]
Good Job!!!

File: testSum.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const add = require('./sum.js');
+
+const testSum = (a, b, c) => {
+  if(add(a,b) === c){
```
Comment (saumya665) [inline testSum.js]: can use double equals sign

**Review by saumya665 — APPROVED** [review]

File: testSum.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const add = require('./sum.js');
+
+const testSum = (a, b, c) => {
```
Comment (allopez7) [inline testSum.js]: Can you change a,b,c into more descriptive names?

**Review by allopez7 — CHANGES_REQUESTED** [review]
Can you change a,b,c into more descriptive names?

File: testSum.js (hunk 2 above)
Comment (seemcat) [reply testSum.js]: Yes!

File: sum.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const add = (a, b) => {
+  return a + b;
```
Comment (allopez7) [inline sum.js]: Change a,b here as well to more descriptive names.

Comment (seemcat) [inline sum.js]: Done!

**Comment by dsope05** [comment]
APPROVED

**Review by dsope05 — APPROVED** [review]

**Review by allopez7 — APPROVED** [review]

**Pull request merged**